src/pattypan/LogManager.java

```diff
@@ -0,0 +1,35 @@
+package pattypan;
+
+import java.io.IOException;
+import java.lang.SecurityException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.logging.ConsoleHandler;
+import java.util.logging.FileHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import java.util.logging.SimpleFormatter;
+
+public class LogManager {
+  public Logger logger = Logger.getLogger(Main.class.getName());
+
+  public LogManager() {
+    try {
+      Path logFileLocation = Paths.get(System.getProperty("user.home") + "/.PattyPan");
```

Comment:
check `getPropertiesDirectiry()` (yeah, there is a typo...) and `readProperties()` in `Settings.java`

Reply:
Nice one, I renamed it and moved it to Utilities as its scope has changed... :-)